pom.xml

```diff
@@ -501,7 +501,7 @@
 		<dependency>
 			<groupId>io.github.openfeign</groupId>
 			<artifactId>feign-gson</artifactId>
-			<version>11.9.1</version>
+			<version>11.10</version>
```

Comment:
was this major version also tested?

Reply:
yes as far as I could.

Reply:
The changes were minor but there were lots of dependency updates to the jar.